zio/zjsonio/stream.go

```diff
@@ -108,7 +108,8 @@ func encodeContainer(typ zng.Type, val []byte) (interface{}, error) {
 				childType = columns[k].Type
```

Comment:
Being that we don't care about aliases at this point I would just on line 94 do `childType = zng.AliasedType(childType) and here: 
```suggestion
				childType = zng.AliasedType(columns[k].Type)
```

Reply:
I don't think I understand this comment as calling AliasedType in the two places you suggest is functionally identical to calling it in the single place that it is currently called?

Reply:
I didn't like the name realType- with your change below this isn't needed.